index.html

```diff
@@ -5,11 +5,49 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="https://unpkg.com/aos@next/dist/aos.css" />
+    <link rel="stylesheet" href="style.css">
     <title>Document</title>
 </head>
 
+
 <body>
-    <h1>My heading</h1>
+    <header>
+        <h1>COLORADO LIFE</h1>
+        <figure><img src="images/god.jpg" alt="god's garden">
+        </figure>
+    </header>
+    <p>Morning</p>
+    <div class=morning>
```

Comment:
Maybe make this a `section` instead? Also, remember to put quotes around your class names:
```suggestion
    <section class="morning">
```
Also, if you do this, consider moving the title "Morning" into an `h2` and inside the `section`, since it's the title of the section.